build

```diff
@@ -16,12 +16,14 @@ cp CodeMirror/lib/codemirror.css out/CodeMirror/lib/codemirror_css.css
 cp CodeMirror/addon/display/autorefresh.js out/CodeMirror/addon/display/autorefresh.js
 cp src/* out
 
+ARCH=$(node -p "os.arch().toUpperCase()")
```

Comment:
I think node's `os.arch()` echoes `uname -m`, which can sometimes disagree with PSoup about the name of an architecture. I have updated the PSoup build to copy the right VM to `out/ReleaseHost`.

Reply:
Have a closer look at node's `os.arch()`, seems that it is determined at compiler time based on host C/C++ compiler, see https://github.com/nodejs/node/blob/122c377eb999a73fd724568c7a3704c13d4c1cfd/configure.py#L1087-L1124